### PR TITLE
Update import project to have friendly names for the extract file organization options

### LINKED ISSUE #10966: Import UX should have friendly names for the enums
image

Repro:
1. Right-click on database in Connections activity
2. Select "Import to Database Project

Expected:
* "Object type" instead of "ObjectType", etc.
* Flavor text for textbox should probably be something like "How do you want the SQL files organized?"

### PR DESCRIPTION
**Pull request by kisantia**
This PR fixes #10966

image

### CODE REVIEW

File: extensions/sql-database-projects/src/controllers/projectController.ts [hunk 1]
```diff
@@ -28,14 +28,13 @@ import { ImportDataModel } from '../models/api/import';
 import { NetCoreTool, DotNetCommandOptions } from '../tools/netcoreTool';
 import { BuildHelper } from '../tools/buildHelper';
 
-// TODO: use string enums
 export enum ExtractTarget {
-	dacpac = 0,
```
Comment (Benjin) [inline extensions/sql-database-projects/src/controllers/projectController.ts:33]: Can this be done in a localizable way?  I know JS enum values are weird, not sure if this is one of those cases.

File: extensions/sql-database-projects/src/controllers/projectController.ts [hunk 2]
```diff
@@ -28,14 +28,13 @@ import { ImportDataModel } from '../models/api/import';
 import { NetCoreTool, DotNetCommandOptions } from '../tools/netcoreTool';
 import { BuildHelper } from '../tools/buildHelper';
 
-// TODO: use string enums
```
Comment (SakshiS-harma) [inline extensions/sql-database-projects/src/controllers/projectController.ts:31]: Thanks for fixing this :).

File: extensions/sql-database-projects/src/controllers/projectController.ts [hunk 3]
```diff
@@ -732,14 +731,11 @@ export class ProjectsController {
 
 		let extractTargetOptions: QuickPickItem[] = [];
 
-		let keys: string[] = Object.keys(ExtractTarget).filter(k => typeof ExtractTarget[k as any] === 'number');
+		let keys: string[] = Object.values(ExtractTarget).filter(k => k !== 'dacpac'); //Do not present the option to create Dacpac
 
 		// TODO: Create a wrapper class to handle the mapping
 		keys.forEach((targetOption: string) => {
-			if (targetOption !== 'dacpac') {		//Do not present the option to create Dacpac
-				let pascalCaseTargetOption: string = utils.toPascalCase(targetOption);	// for better readability
```
Comment (SakshiS-harma) [inline extensions/sql-database-projects/src/controllers/projectController.ts:740]: Can you also remove the util methods and the related test cases for it: toPascalCase, toCamelCase (if still present).

File: extensions/sql-database-projects/src/controllers/projectController.ts [hunk 4]
```diff
@@ -28,14 +28,13 @@ import { ImportDataModel } from '../models/api/import';
...
 
-// TODO: use string enums
 export enum ExtractTarget {
-	dacpac = 0,
-	file = 1,
-	flat = 2,
-	objectType = 3,
-	schema = 4,
-	schemaObjectType = 5
+	dacpac = 'dacpac',
```
Comment (SakshiS-harma) [inline extensions/sql-database-projects/src/controllers/projectController.ts:32]: Just a nit: Though we aren't using "dacpac" anywhere, but better would be to match the case with other enum values.

File: extensions/sql-database-projects/src/controllers/projectController.ts [hunk 5]
```diff
@@ -28,14 +28,13 @@ import { ImportDataModel } from '../models/api/import';
...
+	file = 'File',
+	flat = 'Flat',
+	objectType = 'Object Type',
+	schema = 'Schema',
+	schemaObjectType = 'Schema/Object Type'
```
Comment (SakshiS-harma) [inline extensions/sql-database-projects/src/controllers/projectController.ts:37]: This is pretty good, on the lines of SSDT :).

File: extensions/sql-database-projects/src/controllers/projectController.ts (hunk 3 above)
Comment (kisantia) [reply extensions/sql-database-projects/src/controllers/projectController.ts:740]: removed :)

File: extensions/sql-database-projects/src/controllers/projectController.ts (hunk 1 above)
Comment (kisantia) [reply extensions/sql-database-projects/src/controllers/projectController.ts:33]: Enums can't use computed values, so instead I got rid of the enum and switched to using localized constants.

File: extensions/sql-database-projects/src/controllers/projectController.ts (hunk 4 above)
Comment (kisantia) [reply extensions/sql-database-projects/src/controllers/projectController.ts]: removed it since the enum got removed

**Pull request merged**